src/platform_impl/linux/xkb_keymap.rs

```diff
@@ -0,0 +1,415 @@
+//! Convert Wayland keys to winit keys.
+#![allow(non_upper_case_globals)]
+
+use crate::event::VirtualKeyCode;
+
+/*
+ * TTY function keys, cleverly chosen to map to ASCII, for convenience of
```

Comment:
That looks like yoinked from client-toolkit or something. Could you get rid of C style comments please?

Reply:
Guilty as charged

Reply:
Fixed